osler/core/tests/test_live.py

```diff
@@ -231,8 +217,8 @@ def setUp(self):
         }
 
         self.pt2 = models.Patient.objects.create(
-            first_name="Juggie",
-            last_name="Brodeltein",
+            first_name="Jigie",
+            last_name="Brozeltein",
```

Comment:
haha, what was wrong with the previous name?

Reply:
Lmao I was having this huge problem where because I was using a mix of the old way and factories, both existed in the database.... so there were 2 Juggie Brodelteins in the test db (since that's the name of the PatientFactory one)... so I just changed it so they would be different instead of fixing it LOL sorry